src/app/technical-record/vehicle-summary/vehicle-summary.component.html

```diff
@@ -8,7 +8,10 @@
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
-  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType == 'hgv'">
+  <div
+    class="govuk-summary-list__row"
+    *ngIf="activeRecord.vehicleType === 'hgv' || activeRecord.vehicleType === 'trl'"
```

Comment:
I think now since we have new vehicleType available we need to add one more unit to check that trl section is rendered with activeRecord.vehicleType is trl if not done yet.
We now seem to be having 2 variations of this vehicle-summary.component (one hgv, one trl)
We probably have the 100% line covered but we could add one more that will cover the ngIf.
Thoughts?